base_layer/core/src/transactions/key_manager/inner.rs

```diff
@@ -332,6 +434,54 @@ where TBackend: KeyManagerBackend<PublicKey> + 'static
         Ok((spend_key_id, spend_public_key, script_key_id, script_public_key))
     }
 
+    pub async fn import_key(&self, private_key: PrivateKey) -> Result<TariKeyId, KeyManagerServiceError> {
+        let public_key = PublicKey::from_secret_key(&private_key);
+        let hex_key = public_key.to_hex();
+        self.db.insert_imported_key(public_key.clone(), private_key)?;
+        trace!(target: LOG_TARGET, "Imported key {}", hex_key);
```

Comment:
Possible leak of sensitive information here
```suggestion
       
```

Reply:
it should not, its printing public info, but will remove the log incase